partiql-ast/src/main/kotlin/org/partiql/ast/Ast.kt

```diff
@@ -569,15 +608,28 @@ public object Ast {
         return UpdateTargetStep.Field(key)
     }
 
+    // SFW variant with only required fields; optional fields are set to null
+    @JvmStatic
+    public fun queryBodySFW(select: Select, from: From): QueryBody.SFW {
+        return QueryBody.SFW(select, null, from, null, null, null, null)
+    }
+
+    // SFW variant with `WHERE`; optional fields are set to null
+    @JvmStatic
+    public fun queryBodySFW(select: Select, from: From, where: Expr): QueryBody.SFW {
+        return QueryBody.SFW(select, null, from, null, where, null, null)
+    }
```

Comment:
(self-review) defined the SFW required, SFW required with `WHERE`, and SFW all fields variants since many fields are nullable.

Also the SFW all fields variant is purposely not using `@JvmOverloads`.